### ISSUE TITLE
Quake support removed?

### ISSUE THREAD

**Issue by 0bsidian** [TTimo/GtkRadiant#555]
We are no longer able to configure GtkRadiant for Quake (no longer in the drop down menu at startup). Was support for Quake removed for some reason?

**Comment by TTimo** [comment]
Could be the build process failed to include the game pack, can you confirm? If so, should be a matter of copying the gamepack from the previous zip.

**Comment by TTimo** [comment]
The thing is, the build system does not include a quake1 game pack. I found this in my email archive: https://github.com/TTimo/GtkRadiant/pull/290 - maybe you were just dragging along an old gamepak.

@ericwa are you still around? Should we be finishing the quake1 support or ditching it completely? I'm not sure how 'quake 1' got listed in our supported gamepacks when it was never completed to begin with.

**Comment by ericwa** [comment]
Hey, still around - sorry for dropping the ball on this.

I just tested it on the 1.6.5 release, and the gamepack I posted in #290 works. Here is the link again: https://www.dropbox.com/s/uj8iewxy2q7ozmt/Q1Pack.zip?dl=1

I haven't used it much but it the essentials seem to work - loading/saving maps, texturing, adding entities (no model previews, but that's not a showstopper), so would it be worth adding my gamepack to SVN?

**Comment by 0bsidian** [comment]
I've had people asking me about Q1 support, in particular when people are working on stuff like this:

https://youtube.com/watch?v=K8JWgsz8-lQ&feature=youtu.be

**Comment by TTimo** [comment]
That's hot.

Ok let's see if we can finish this up then. Looking at the old emails I think the expectation was that I would be the one putting in the new Q1 gamepak, which I never got around to do and somehow the website started saying we had it done.

**Comment by TTimo** [comment]
I have included the Q1 pack manually into the latest 1.6.5 build. But let's keep this open until we have this better supported.

**Comment by AraHaan** [comment]
Does this support The Ultimate Doom well too? I would like to get a much better Doom Editor that does not crash on Windows 7 Ultimate x64. 👀

**Comment by 0bsidian** [comment]
No, Doom predates all versions of Radiant, there has never been support for Doom.

**Comment by 0bsidian** [comment]
For a list of supported games, see here:
http://icculus.org/gtkradiant/developers.html#gamepacks

**Comment by Garux** [comment]
@0bsidian
since there are people, asking about Q1 support, you might find this [radiant](https://github.com/Garux/netradiant-custom/releases) useful
It's showing models, has verified entities definitions, valve220 mapformat and even "Stupid Quake Bug" support .o

**Comment by TTimo** [comment]
The Q1Pack is enabled (next build). Would be nice to have a sample map though? Anyone got something to contribute?

**Comment by jdolan** [comment]
I mean.. the originals are all GPL'ed so why not start.map?

**Comment by 0bsidian** [comment]
John Romero released them a few years ago, download link here:
http://rome.ro/s/quake_map_source.zip

**Comment by TTimo** [comment]
I'm adding the maps to the gamepak, and fixed an issue where it would not load .MAP files (only .map).
Looks pretty broken still, not finding any textures:
`WARNING: could not find "TECH08_1" in any listed wad files, searching all wad files instead!
WARNING: could not find "TECH08_1" in the vfs search path`

**Comment by ericwa** [comment]
@TTimo in: https://github.com/TTimo/GtkRadiant/pull/290 I listed a bunch of requirements for textures to load under: "Some user documentation", maybe one of those points is causing your textures not to load

**Comment by TTimo** [comment]
But the texture wads are in PAK0.PAK / PAK1.PAK aren't they? Does this imply that it doesn't know how to read in the pak files atm?

**Comment by ericwa** [comment]
In Quake's PAK0.PAK/PAK1.PAK, the textures are embedded in each .bsp file (so there are redundant copies of textures), and there are no .wad files.

One map editor (QuArK) could read the textures from the bsp's in the .pak files directly, but almost all other editors (and all Quake compile tools) require the mapper to provide their own .wad files with the textures they want to use.

For testing, you can get a Q.wad from https://www.quaddicted.com/files/wads/q.zip containing the base game + expansion's textures.

**Comment by TTimo** [comment]
Ok .. should we / could we distribute a basic .wad to help with this?
